### ISSUE TITLE
Game crashes with latest OBS.

### ISSUE THREAD

**Issue by CortexReaver** [FWGS/xash3d-fwgs#1870]
How to reproduce:
1. Set "capture any fullscreen application" in OBS
2. run Xash and then go to "Custom game" as fast as possible, before OBS manages to capture a screen (a preview screen should be black)
3. when you see picture in OBS review, try to click on some of the mods. Game just hangs there.

**Comment by mittorn** [comment]
Looks like some obs bug. Try run with debugger and look where it hangs

**Comment by CortexReaver** [comment]
Debugger of what? OBS or Xash? How to run debugging on Xash?

**Comment by mittorn** [comment]
Attach debugger to xash after OBS injected. If it injects some code that hangs, you possibly will see it in debugger stack trace

31 октября 2024 г. 1:15:01 GMT+03:00, CortexReaver ***@***.***> пишет:
>Debugger of what? OBS or Xash? How to run debugging on Xash?

**Comment by CortexReaver** [comment]
Oh, I see what you mean. I am not a programmer nor I have any IDE of some sort. I thought there are some command that I run Xash binary with and it makes a log file that I can attach to a message here.

**Comment by mittorn** [comment]
Ok, someone with dev environment need to reproduce this. Can you 
describe exact OBS platform and capture plugin used?

**Comment by CortexReaver** [comment]
Yes, read first post please.

**Comment by CortexReaver** [comment]
UPD: I've deleted all unnecessary OBS plugins and seems like all the stutters all gone. Not sure about crashes though.

**Comment by a1batross** [comment]
@SNMetamorph could you check it?

**Comment by SNMetamorph** [comment]
> @SNMetamorph could you check it?

I tried, but fullscreen mode itself works very weird. Mouse cursor position being mismatched after going to fullscreen, and then game just stopped to running with such error
![image](https://github.com/user-attachments/assets/3d00819a-f297-489e-93df-e3ac4c2da73b)

**Comment by SNMetamorph** [comment]
Seems that video modes switching are broken

**Comment by a1batross** [comment]
.. again? D:

It's true fullscreen, not borderless, right?

**Comment by SNMetamorph** [comment]
Yeah, exactly fullscreen, not borderless. I even can't test OBS with it. But with windowed mode, OBS worked flawlessly, even when switching mods.
